aqua-examples/near-integration/README.md

```diff
@@ -3,41 +3,9 @@
 
-## Tooling
+In our examples we've been using the [Aqua CLI](https://doc.fluence.dev/aqua-book/aqua-cli) `aqua` and [Marine tooling](https://doc.fluence.dev/marine-book/marine-tooling-reference) (the [Marine REPL](https://doc.fluence.dev/marine-book/marine-tooling-reference/marine-repl) `mrepl` and [Marine CLI](https://doc.fluence.dev/marine-book/marine-tooling-reference/marine-cli) `marine`).
 
-In our examples we're using the Aqua CLI `aqua` and [Marine tooling](https://doc.fluence.dev/marine-book/marine-tooling-reference) (the [Marine REPL](https://doc.fluence.dev/marine-book/marine-tooling-reference/marine-repl) `mrepl` and [Marine CLI](https://doc.fluence.dev/marine-book/marine-tooling-reference/marine-cli) `marine`).
-
-The Aqua CLI `aqua` enables management of Aqua development and includes:
-
-- **Compiler** converting Aqua code to either JavaScript/TypeScript handlers wrapping Aqua Intermediary Representation (AIR), or pure AIR;
-
-- **Client peer**, a one-shot client peer over the compiled Aqua code specified allowing you to quickly and effectively test Aqua code against deployed services on the network;
-
-- **Utilities** helping in various development and operations workflows such as a keypair creation (ed25519-based), module distribution (deploying and removing services), environment listing showsing a list of peers for a specific Fluence network (`krasnodar`, `stage` or `testnet`).
-
-The Marine tooling `mrepl` and `marine`:
-
-- **Marine REPL** which wraps a local Marine runtime to allow developers to interact with Wasm module interfaces and business logic locally;
-
-- **Marine CLI** allows developers to build Marine modules and examine various info.
-
-The Fluence CLI `fluence` aims to provide you with the improved unified developer experience while having the same functionality available with the Aqua CLI and Marine tooling:
-
-- Complile Aqua code;
-
-- Project generation for an easy start;
-
-- Build and debug Marine modules using REPL;
-
-- Download compiled WASM modules;
-
-- Deploy and remove services;
-
-- Aqua code wrapper generation for deployed services;
-
-- Setup environment by installing all the prerequisites and dependencies (Aqua and Marine tooling, Rust, WASM compile target)
-
-Under the hood the Fluence CLI leverages all the tooling available (both Aqua and Marine).
+For the purpose of this tutorial, we'll be using Fluence's new `fluence` CLI tool, which unifies the CLIs you have already been using, e.g., the `aqua` CLI and Marine tooling CLIs (`marine` and `mrepl`), and brings additional features such as project template generation, wrapper generation for deployed services, project dependencies install. See the [Fluence CLI docs](https://github.com/fluencelabs/fluence-cli#readme) for more information.
```

Comment:
it's more of a wrapper than a unifying interface, isn't it?

Reply:
yeah, it's more precise, at least for now. I'll change `unifies` to `wraps`.